src/components/settings/SettingsSetLogLevel.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+	<div>
+		<fieldset>
+			<legend>{{ t('logreader', 'Set backend loglevel') }}</legend>
+			<NcCheckboxRadioSwitch v-for="levelName, levelId in LOGGING_LEVEL_NAMES"
+				:key="levelId"
+				:checked="logLevel"
+				:value="`${levelId}`"
+				type="radio"
+				name="loglevel"
+				@update:checked="setLogLevel">
+				{{ levelName }}
+			</NcCheckboxRadioSwitch>
+		</fieldset>
+	</div>
+</template>
+
+<script setup lang="ts">
+import type { IAppSettings } from '../../interfaces'
+
+import { computed } from 'vue'
+import { showError } from '@nextcloud/dialogs'
+import { translate as t } from '@nextcloud/l10n'
+import { useSettingsStore } from '../../store/settings'
+import { LOGGING_LEVEL_NAMES } from '../../constants'
+
+import NcCheckboxRadioSwitch from '@nextcloud/vue/dist/Components/NcCheckboxRadioSwitch.js'
+
+const settingsStore = useSettingsStore()
+
+/**
+ * Get currently set loglevel
+ */
+const logLevel = computed(() => `${settingsStore.logLevel}`)
+
+const setLogLevel = (level: string) => {
+	const numericLevel = parseInt(level) as IAppSettings['logLevel']
+	settingsStore.setSetting('logLevel', numericLevel)
+		.catch(() => showError(t('logreader', 'Could not set logging level')))
```

Comment:
```suggestion
	try {
		settingsStore.setSetting('logLevel', numericLevel)
	} catch (e) {
		logger.error(e as Error)
		showError(t('logreader', 'Could not set logging level'))
	}
```

I know we use that pattern in other places but I would recommend we log the catched errors for debugging any problems. (logger imported from `utils/logger`

Reply:
Done via `catch (e => { ... })` because `setSetting` gives us a promise

Reply:
You could also await the promise, but fine with me :)